.github/workflows/ci.yml

```
@@ -0,0 +1,28 @@
+name: Continuous Integration
+
+on:
+  - push
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 5
+
+      - name: Setup Node.JS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "npm"
+      - name: Check Dependencies
```

nit; add spacing here
```suggestion
          cache: "npm"
          
      - name: Check Dependencies
```